debezium-connector-db2/src/main/java/io/debezium/connector/db2/Db2ConnectorConfig.java

```diff
@@ -302,18 +305,21 @@ public static ConfigDef configDef() {
         Field.group(config, "DB2 Server", HOSTNAME, PORT, USER, PASSWORD, SERVER_NAME, DATABASE_NAME, SNAPSHOT_MODE);
         Field.group(config, "History Storage", KafkaDatabaseHistory.BOOTSTRAP_SERVERS,
                 KafkaDatabaseHistory.TOPIC, KafkaDatabaseHistory.RECOVERY_POLL_ATTEMPTS,
-                KafkaDatabaseHistory.RECOVERY_POLL_INTERVAL_MS, HistorizedRelationalDatabaseConnectorConfig.DATABASE_HISTORY);
-        Field.group(config, "Events", RelationalDatabaseConnectorConfig.TABLE_WHITELIST,
-                RelationalDatabaseConnectorConfig.TABLE_BLACKLIST,
-                RelationalDatabaseConnectorConfig.COLUMN_BLACKLIST,
-                RelationalDatabaseConnectorConfig.SNAPSHOT_SELECT_STATEMENT_OVERRIDES_BY_TABLE,
-                RelationalDatabaseConnectorConfig.TABLE_IGNORE_BUILTIN,
+                KafkaDatabaseHistory.RECOVERY_POLL_INTERVAL_MS, Db2ConnectorConfig.DATABASE_HISTORY);
+        Field.group(config, "Events", Db2ConnectorConfig.TABLE_WHITELIST,
+                Db2ConnectorConfig.TABLE_INCLUDE_LIST,
+                Db2ConnectorConfig.TABLE_BLACKLIST,
+                Db2ConnectorConfig.TABLE_EXCLUDE_LIST,
+                Db2ConnectorConfig.COLUMN_BLACKLIST,
+                Db2ConnectorConfig.COLUMN_EXCLUDE_LIST,
+                Db2ConnectorConfig.SNAPSHOT_SELECT_STATEMENT_OVERRIDES_BY_TABLE,
+                Db2ConnectorConfig.TABLE_IGNORE_BUILTIN,
```

Comment:
Beside adding the new include/exclude list properties I did some housekeeping here to move the parameters to the ConnectorConfig class related to that connector in order to local rewrites work and from architectural perspective we have less dependencies to "the outside".